server/src/mindustry/server/ServerControl.java

```diff
@@ -269,6 +270,27 @@ protected void setup(String[] args){
 
             info("Server loaded. Type @ for help.", "'help'");
         });
+
+        Events.on(PlayerJoin.class, e -> {
+            if(state.serverPaused && autoPaused && Config.autoPause.bool()){
+                state.serverPaused = false;
+                autoPaused = false;
+
+                info("Game was automatically resumed because a player joined.");
```

Comment:
This is going to be logged a lot, is it really particularly important to write about either of these events? The server already logs when someone joins/leaves.

Reply:
Probably this should be included in the join/leave messages, ex. `Player has joined/left the game, resuming/pausing the game`